src/index.html

```diff
@@ -20,8 +20,407 @@
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#">
+          <img
+            src="images/logo.png"
+            alt="logo.png"
+            class="logo__img"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="image"
+            src="images/imac.jpeg"
+            alt="imac"
+          />
+          <div class="content-wrapper">
+            <div class="product-name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </div>
+            <div class="product-code">Product code: 195434</div>
+            <div class="content-revie">
+              <div class="stars">
+                <div class="stars stars--4">
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                </div>
+              </div>
+              <div class="revie-text">Reviews: 5</div>
+            </div>
+            <div class="content-price">
+              <div class="price-text">Price:</div>
+              <div class="prise-num">$2,199</div>
```

Comment:
There is a typo in the class name `prise-num`. It should be `price-num` to maintain consistency and clarity.